### PR TITLE
Disable horizontal resizing for textarea across hq

### PR DESCRIPTION
**Pull request by jingcheng16**
## Product Description

Original Ticket: https://dimagi-dev.atlassian.net/browse/SAAS-13281

After consulting other's opinion in #gtd-product, instead of implementing what the original ticket ask for (resize both vertically and horizontally), I disabled horizontally resizing for all textarea.
Related slack discussion: 
https://dimagi.slack.com/archives/C0BGN9LDU/p1658431905764939
https://dimagi.slack.com/archives/C0BGN9LDU/p1658477079788099

Here is a google doc I made to track almost every changes:
https://docs.google.com/document/d/1Hs9oDE_RlX2DmzlriuaQo-xGPX7EGoendUpcd29lRVU/edit?usp=sharing

## Technical Summary

I searched two patterns in our hq codebase:  `<textarea(.|\n)*form-control`, `widget=forms.Textarea` , to find all the textarea might need change. And I either add `vertical-resize` to its class, or add `"style": "resize:vertical"` to its attributes.

## Feature Flag

This is not specific to a feature flag. But many feature flags might need to toggle on to see the changes.

### Safety story

I find almost every changes I made.
Also, there is no functionality changed. Just disabling horizontal resizing should be safe.
This change aligns with our [style guide](https://www.commcarehq.org/styleguide/organisms/#:~:text=The%20textarea%20uses,the%20page%27s%20layout)

### QA Plan

Talked with Jenny, no QA for this one. 

### Rollback instructions

- [X] This PR can be reverted after deploy with no further considerations

### Labels & Review
- [X] Risk label is set correctly
- [X] The set of people pinged as reviewers is appropriate for the level of risk of the change

### CODE REVIEW

File: corehq/apps/accounting/forms.py [hunk 1]
```diff
@@ -841,7 +841,7 @@ class ChangeSubscriptionForm(forms.Form):
     subscription_change_note = forms.CharField(
         label=gettext_lazy("Note"),
         required=True,
-        widget=forms.Textarea,
+        widget=forms.Textarea(attrs={"style": "resize:vertical"}),
```
Comment (orangejenny) [inline corehq/apps/accounting/forms.py:844]: CSS classes are preferable to inline styles - could this and similar code be `attrs={'class': 'vertical-resize'}` instead?

File: corehq/apps/domain/forms.py [hunk 2]
```diff
@@ -1776,10 +1776,12 @@ def save(self, commit=True):
 class ProBonoForm(forms.Form):
     contact_email = MultiCharField(label=gettext_lazy("Email To"), widget=forms.Select(choices=[]))
     organization = forms.CharField(label=gettext_lazy("Organization"))
-    project_overview = forms.CharField(widget=forms.Textarea, label="Project overview")
+    project_overview = forms.CharField(widget=forms.Textarea(attrs={"style": "resize:vertical"}),
```
Comment (orangejenny) [inline corehq/apps/domain/forms.py:1779]: Nitpick: weird spacing, it looks like `label` is an attribute of the form. It's more common to put the params on their own line:
```
project_overview = forms.CharField(
    widget=forms.Textarea(attrs={"style": "resize:vertical"}), label="Project overview"
)
```
or to line up the kwargs:
```
project_overview = forms.CharField(widget=forms.Textarea(attrs={"style": "resize:vertical"}),
                                   label="Project overview")
```

**Comment by jingcheng16** [comment]
@orangejenny 
Hi Jenny, I changed to inline style and the spacing in this commit: 936114515aab480c7e8f7f2b2e78ac97dfe76ddd
Please review

File: corehq/apps/enterprise/forms.py [hunk 3]
```diff
@@ -31,7 +31,7 @@ class EnterpriseSettingsForm(forms.Form):
         label="Signup Restriction Message",
         required=False,
         help_text=gettext_lazy("Message to display to users who attempt to sign up for an account"),
-        widget=forms.Textarea(attrs={'rows': 2, 'maxlength': 512}),
+        widget=forms.Textarea(attrs={'rows': 2, 'maxlength': 512, "style": "resize:vertical"}),
```
Comment (orangejenny) [inline corehq/apps/enterprise/forms.py:34]: missed one

**Review by orangejenny — APPROVED** [review]
Flagged one widget that's still using inline styling - once that's updated, this is good to merge.

**Review by orangejenny — APPROVED** [review]
beautiful

**Pull request merged**